### PR TITLE
Update wallet-data.ts - phantom

### LINKED ISSUE #13744: Suggest a wallet
### Wallet name

Phantom

### What type of wallet?

EOA

### Wallet description

The friendly crypto wallet built for DeFi & NFTs.

### Wallet logo

Phantom-Logo-Purple
Phantom-Icon_App

### Background color for brand logo

#AB9FF2

### URL to the project

https://phantom.app/

### When did the wallet go live to users?

May 17th 2021

### What are the fees users may be paying for using your wallet product?

0.85% fee for in-app swaps

### Does the wallet have an active development team?

Yes

### Is the wallet globally accessible?

Yes, see terms for jurisdictions that are excluded: https://phantom.app/terms

### Is the wallet available in multiple languages?

English, Spanish, German, French, Italian, Chinese, Bangla, Japanese, Korean, Russian, Hindi, Indonesian, Malay, Thai, Vietnamese, Portuguese, Turkish, Filipino, Burmese, Amharic, Arabic, Gujarati, Hausa, Igbo, Punjabi, Swahili, Tamil, Telugu, Yoruba

### What social links are there for the project?

https://x.com/phantom
https://www.reddit.com/r/Phantom/
https://www.linkedin.com/company/phantomwallet
https://www.youtube.com/@phantom-wallet

### Does the wallet have a mobile app? If yes, which operating systems are supported (iOS, Android)?

iOS: https://apps.apple.com/us/app/phantom-crypto-wallet/id1598432977
Android: https://play.google.com/store/apps/details?id=app.phantom&hl=en_US&pli=1

### Does the wallet have a desktop app? If yes, which operating systems are supported (Windows, Mac, Linux)?

N/A

### Does the wallet have a browser extension? If yes, which browsers are supported (Chromium, Firefox, Safari)?

Chrome: https://chromewebstore.google.com/detail/phantom/bfnaelmomeimhlpmgjnjophhpkkoljpa
Firefox: https://addons.mozilla.org/en-US/firefox/addon/phantom-app/

### Is it a hardware wallet?

No, but Phantom supports Ledger devices.

### What is the availability of your source code?

Closed source

### What license is the wallet software released under?

N/A

### Who holds the private keys?

Phantom is a self-custody wallet. Users have access to their public and private keys.

### Please describe the measures taken to ensure the wallet's security and provide documentation wherever possible

Phantom employs a full-time, dedicated in-house security team. The app is independently audited and offers a bug bounty with rewards up to $50,000. For more info please visit https://phantom.app/security.

Audits: https://github.com/phantom/audit-reports
Bug Bounty: https://phantom.app/bug-bounty

### Has the wallet's smart contract code or security modules been audited?

https://github.com/phantom/audit-reports

### Does the wallet have an internal security team?

Yes. https://phantom.app/security

### Any other security testing that should be noted?

See https://phantom.app/security

### Scam protection?

Yes. Phantom has a dedicated blocklist for scam domains and malicious tokens: https://github.com/phantom/blocklist

Phantom also provides human readable simulation results on all transaction and message signing requests. If a request is deemed malicious, Phantom will show warnings and/or block the request.

Phantom also has advanced spam filters for hiding unwanted or malicious tokens.

### Does the wallet support connecting to Ethereum applications?

Yes https://docs.phantom.app/

### Does the wallet support connecting to a hardware wallet?

Yes

### Does the wallet support importing Ethereum RPC endpoints?

No

### Does the wallet support viewing and interacting with NFTs?

Yes

### Does the wallet support staking directly?

Currently Solana only https://help.phantom.app/hc/en-us/articles/4406374138771--Staking-Solana

### Does the wallet support swaps directly?

Yes https://help.phantom.app/hc/en-us/articles/6048249796243-Phantom-Swapper-FAQ

### Does the wallet support bridging directly?

Yes https://help.phantom.app/hc/en-us/articles/23226373773971-Cross-Chain-Swapper

### Does the wallet support multi-chain networks?

Yes https://help.phantom.app/hc/en-us/articles/4406393831187-How-to-deposit-SOL-ETH-MATIC-and-Bitcoin-in-my-Phantom-wallet

### Which Ethereum Layer 2 networks does the wallet support?

Not currently

### Does the wallet allow the user to customize gas fees?

Yes https://help.phantom.app/hc/en-us/articles/10500478712083-Why-is-my-transaction-still-pending

### Does the wallet support sending transactions to ENS addresses?

Yes https://help.phantom.app/hc/en-us/articles/16709877045651-How-to-send-tokens-using-domain-names

### Does the wallet support importing or automatically querying and displaying ERC-20 tokens?

Phantom automatically queries and displays all ERC-20s https://help.phantom.app/hc/en-us/articles/27309470600851-How-do-I-add-a-token

### Does the wallet support EIP-1559 (type 2) transactions?

Yes

### Does the wallet have any recovery options?

Phantom supports recovering wallets via seed phrase or private keys https://help.phantom.app/hc/en-us/articles/25334064171795-How-to-view-your-Secret-Recovery-Phrase-or-Private-Keys

### Does the wallet have fiat on-ramps through credit/debit cards, wire transfers, or bank transfers (ACH)?

Yes https://help.phantom.app/hc/en-us/articles/4406543783571-How-to-buy-Solana-SOL-Ethereum-ETH-Polygon-MATIC-and-Bitcoin-BTC

### Does the wallet support withdrawals to fiat?

Not currently

### Is the wallet a multi-signature wallet?

No

### Does the wallet support social recovery?

No

### Does the wallet support bundled transactions?

No

### Who can the ethereum.org team can contact regarding the wallet in future?

@brianfriel on Telegram

### Does the wallet have a dedicated support team?

Yes. Phantom has a dedicated support staff responds to all requests. Users can reach out via the support link in app or via https://help.phantom.app/hc/en-us

### What educational resources/documentation do you provide to users?

Educational resources: https://phantom.app/learn
Developer resources: https://docs.phantom.app/
Support resources: https://help.phantom.app/hc/en-us

### Does the wallet have any integrated tools not mentioned above?

_No response_

### Would you like to work on this issue?

- [ ] Yes
- [X] No

### PR DESCRIPTION
**Pull request by konopkja**
## Related Issue

- Closes #13744

### CODE REVIEW

**Comment by wackerow** [comment]
Erroring because of the language codes that were added (`ha` and `yo`, both supported languages) are not yet added to the `Lang` type required here.

#14269 Fixes this; blocking this PR briefly until that PR is merged.

**Review by corwintines — APPROVED** [review]

**Comment by corwintines** [comment]
@wackerow merged that PR in, so going to merge this. Thanks for that!

**Pull request merged**
